File: Semester_2/Betriebs_und_Kommonikationssysteme_SoSe25/Tutorium/fibunacci.c
#include <stdio.h>

int fibo(int zahl);

int main(){
  int fib_zahl = 8;

  for (int i = 0; i <= fib_zahl; i++){
    printf("fib(%d) = %d\n",i,fibo(i));
  }

  return 0;
}

int fibo(int zahl){

  if (zahl <= 1){
    return zahl;
  }
  else{
    return fibo(zahl-1) + fibo(zahl-2);
  }
}

File: Semester_2/Algorithmen und Datenstrukturen/Tutorium/fibunacci.scala
import scala.io.StdIn.readLine


def fibo(zahl: Int): Int =
  if zahl <= 1 then
    return zahl
  else
    return fibo(zahl - 1) + fibo(zahl - 2)


@main def start(): Unit = 
  println("Bitte zahl eingeben: ")
  var input = readLine().toInt

  for i <- 0 to input.toInt do 
    print(s"f($i): " + fibo(i) + "\n")


